### DISCUSSION TITLE
Time to flag crouton as uninstallable for current releases

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5099] [General]
@dnschneid @DennisLfromGA @drinkcat 

For me, crouton has been broken ever since 126.0.6478.252. If this holds true in general, then no one will be able to install crouton who has an up-to-date Chromebook. Since David is currently using my "How To", located at

[Community Supplied Instructions](https://github.com/dnschneid/crouton/wiki/Updated-Installation-Instructions-for-Crouton)

I have edited the document with a warning that attempting to install crouton may be a waste of time. We want to save people the frustration of running into a wall and wasting their time trying to install crouton, when it's never going to work.

Furthermore, we don't need any more "issues" being reported, because there appears to be no fix for this.

Until, and unless, one of the developers is able to fix this issue, we need to warn people off from attempting to install crouton. We seem to have reached the end of the line where crouton is concerned.

**Comment by dnschneid** [comment]
related: #5097 
thanks for keeping on top of the docs, and yeah, a warning is probably in order.

**Reply by CroutonIsFun** [reply]
I really miss you! Thanks for all you've done. It's been a great run.

**Comment by CroutonIsFun** [comment]
@dnschneid @DennisLfromGA @drinkcat 

The situation is worse than I thought. Attempting to install crouton will now corrupt your Chromebook, requiring a powerwash to set things back to normal. I've done this twice now, and each time I've had to reset my Chromebook.

**Comment by SkoolerK** [comment]
I was able to install crouton on 129.0.6668.99 on an Acer Spin 514 using a slightly modified version of your how to. I had an issue with the file naming convention for the first command, where instead of sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton, I had better luck with sudo install -Dt /usr/local/bin -m 755 ~/MyFiles/Downloads/crouton. Thank you all for your dedication to keep it running.

**Reply by CroutonIsFun** [reply]
I'm glad you had success. The mod seems to be the same effect as the original instructions. This is very confusing to me.

**Reply by EzArclight** [reply]
question @CroutonIsFun are you saying that the change did not work for you? or that it did? I cannot tell by your wording.

**Reply by CroutonIsFun** [reply]
Sorry, no, the location of the downloads directory has changed. Your work around is correct. I discovered that you can install crouton on ARM devices, but not AMD. At least not for current releases.

**Comment by EzArclight** [comment]
I have also gotten crouton to work on a n23 yoga using the method proposed by @SkoolerK and on 130 no less, so I think that fixes it.

**Reply by CroutonIsFun** [reply]
I don't see anything at that link.
